requirements.txt

```diff
@@ -17,3 +17,4 @@ yuicompressor==2.4.8
 Flask-Admin==1.1.0
 Flask-Login==0.2.11
 WTForms==2.0.2
+Flask-Mail==0.9.1
```

Comment:
Are these really environment variables, or just global Python variables? If I set the env var MAIL_USERNAME, will it not take the one you defined here?